cmd/expiration-mailer/main.go

```diff
@@ -367,14 +367,13 @@ func main() {
 			log:           logger,
 			dbMap:         dbMap,
 			rs:            sac,
-			mailer:        &mailClient,
+			mailer:        mailClient,
```

Comment:
Yup, now that `mail.Mailer` is an interface and `mail.New` returns `*MailerImpl` this shouldn't be a reference.